flipperdevices/flipperzero-firmware
# SubGhz: Add new man key

# What's new

- SubGhz: fix kelog pattern
- SubGhz: add Keelog Secure Learning
- SubGhz: add man NICE_Smilo, NICE MHOUSE, DEA, Genius, FAAC RC,XT, CAME Space, DTM

# Verification 

- Update in qFlipper, with Asset update.  
- Run the SubGhz application and check the reception of these protocols

# Checklist (do not modify)
